src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private static final Random random = new Random(47);
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        return new Ball(Color.valueOf(colorSupplier.getRandomColor()), random.nextInt(101));
```

Comment:
The magic number 101 used in `random.nextInt(101)` should be replaced with a named constant to improve code readability and maintainability. Define it as a private static final field with an appropriate name.